Dockerfile

```diff
@@ -5,3 +5,11 @@ RUN pip install numpy pandas flask pygal smalluuid zipstream python-dateutil git
 WORKDIR /app
 
 ADD . /app
+
+RUN pip install -r dev-requirements.txt
```

Comment:
Could we collapse all of these RUN layers into one?
```
RUN pip install -r dev-requirements.txt \
    && pip install -r tox-requirements.txt \
    && ...
```

This way we can minimize the number of layers in our image. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
Done.